console.rst

```diff
@@ -38,12 +38,12 @@ want a command to create a user::
         // the name of the command (the part after "bin/console")
         protected static $defaultName = 'app:create-user';
 
-        protected function configure()
+        protected function configure(): void
```

Comment:
@wouterj shall we create a DOCtor rule to ensure all method declarations have a return type?

Reply:
I'm afraid we can't. E.g. we cannot yet use union types in the docs, nor does `self` work in all cases (in most cases, it only works as expected in PHP 7.4+). Also, we have to be careful about not conflicting with the core code.